External/SPMmodified/xASL/xASL_im_FillNaNs.m

```diff
@@ -6,12 +6,9 @@
 %   InputPath   - path to image, or image matrix (REQUIRED)
 %                 NB: if this is a path, it is overwritten
 %   UseMethod   - one of the following methods to fill the NaNs:
-%                 1: extrapolate by smoothing (useful for any image, e.g.
-%                    extrapolating M0 smooth biasfield) (DEFAULT)
-%                 2: set to 0 (useful for e.g. DARTEL displacement fields,
-%                    where 0 means no displacement)
-%                 3: linearly extrapolate 
-%                 4: fill with flow field identity map (specific to MNI 1.5mm space used by CAT12/DARTEL)
+%                 1: extrapolate by smoothing (useful for any image, e.g. extrapolating M0 smooth biasfield) (DEFAULT)
+%                 2: set to 0 (useful for e.g. DARTEL displacement fields, where 0 means no displacement)
+%                 3: extrapolate deformation field linearly - this method is adapted to SPM deformation fields XxYxZx3 
```

Comment:
should be XxYxXx1x3

Reply:
Good point, but only party true. Inside cat, it is XxYxZx3, so we need both options, I've fixed the code to work with this.

Reply:
Nice